src/main/Config.h

```diff
@@ -234,15 +234,29 @@ class Config : public std::enable_shared_from_this<Config>
     // core can process simultaneously. If core is at capacity, it temporarily
     // stops reading from a peer until it completes processing of at least one
     // in-flight message
-    uint32_t PEER_READING_CAPACITY;
+    int32_t PEER_READING_CAPACITY;
 
     // A config parameter that controls how many flood messages (tx or SCP) from
     // a particular peer core can process simultaneously
-    uint32_t PEER_FLOOD_READING_CAPACITY;
+    int32_t PEER_FLOOD_READING_CAPACITY;
```

Comment:
Can these  have negative values? Or do we want to keep it same as all others, int instead of uint?